cleanlab/datalab/internal/issue_manager/label.py

```diff
@@ -86,11 +86,11 @@ def __init__(
     def _process_find_label_issues_kwargs(**kwargs: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
We can omit the type hint when the `kwargs` dictionary gets unpacked:

```suggestion
    def _process_find_label_issues_kwargs(**kwargs) -> Dict[str, Any]:
```